lib/radiator_web/router.ex

```diff
@@ -9,6 +9,18 @@ defmodule RadiatorWeb.Router do
     plug :put_secure_browser_headers
   end
 
+  pipeline :public_browser do
+    plug :accepts, ["html", "xml", "rss"]
+    plug :fetch_session
```

Comment:
Yeah - good point. We probably should. When I tried it blew up in my face though (probably just because the use of the flash in the app template). Lets make this a follow up.

Reply:
Did you remove all three? That worked for me. `:protect_from_forgery` relies on sessions, so if you only removed sessions and/or flashes (which is what I did) it blows up.